src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+
+        UserService userService = new UserService();
+        if (userService.findByEmail(email) == null) {
+            return false;
+        } else {
+            return email.equals(userService.findByEmail(email).getEmail())
+                    && password.equals(userService.findByEmail(email).getPassword());
```

Comment:
Why do you need to call the method to get the user every time you need to check sth? You can simplify it here 
```suggestion
        User user = userService.findByEmail(email);
        if (user != null && user.getPassword().equals(password)) {
          return true;
        }
        return false;
```
OR
```suggestion
        User user = userService.findByEmail(email);
        return user != null && user.getPassword().equals(password).
```